deployment/v3/mosip/admin/install.sh

```
@@ -35,7 +36,7 @@ function installing_admin() {
   helm -n $NS install admin-service mosip/admin-service --set istio.corsPolicy.allowOrigins\[0\].prefix=https://$ADMIN_HOST --wait --version $CHART_VERSION
 
   echo Installing admin-ui
-  helm -n $NS install admin-ui mosip/admin-ui --set admin.apiUrl=https://$API_HOST/v1/ --set istio.hosts\[0\]=$ADMIN_HOST --version $CHART_VERSION
+  helm -n $NS install admin-ui mosip/admin-ui --set admin.apiUrl=https://$API_HOST/v1/ --set istio.hosts\[0\]=$ADMIN_HOST --version $ADMIN_UI_CHART_VERSION
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)